co2calculator/distances.py

```diff
@@ -544,90 +540,51 @@ def get_distance(request: DistanceRequest) -> Kilometer:
     :rtype: Kilometer
     """
 
-    detour_map = {
-        TransportationMode.CAR: False,
-        TransportationMode.MOTORBIKE: False,
-        TransportationMode.BUS: True,
-        TransportationMode.TRAIN: True,
-        TransportationMode.TRAM: True,
-        TransportationMode.PLANE: True,
-        TransportationMode.FERRY: False,
-        TransportationMode.BICYCLE: False,
-        TransportationMode.PEDELEC: False,
-    }
+    # Calculate cords
+    coords = []
+    # StructuredLocation, TrainStation, Airport based on the object class of
+    # start and destination in the request
+    for loc in [request.start, request.destination]:
+        if loc.__class__ is StructuredLocation:
```

Comment:
```suggestion
        if isinstance(loc, StructuredLocation):
```
I think that's the usual way to check for the class of an object